src/mui/auth/Login.js

```diff
@@ -70,7 +71,7 @@ class Login extends Component {
     login = ({ username, password }) => {
         const { authClient, push, location } = this.props;
         if (!authClient) return;
-        authClient(AUTH_LOGIN, { username, password })
+        return authClient(AUTH_LOGIN, { username, password })
```

Comment:
If we follow the `redux-form` doc, the `onSubmit` function must return a promise to handle the `submit` state of the form. This allow usage of the `submitting` props in [Login.js](https://github.com/Natsuke/admin-on-rest/blob/2eff497dfa9c62fb4c86f48af0d74fe747309480/src/mui/auth/Login.js#L80)

cfr: http://redux-form.com/6.5.0/docs/api/ReduxForm.md/#-submit-promise-